internal/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,140 @@
+package healthcheck
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"github.com/chia-network/go-chia-libs/pkg/rpc"
+	"github.com/chia-network/go-chia-libs/pkg/types"
+)
+
+// Healthcheck is the main container for the app
+type Healthcheck struct {
+	healthcheckPort uint16
+	client          *rpc.Client
+
+	// Last block height we received
+	lastHeight uint32
+
+	// Time we received the last block height
+	lastHeightTime time.Time
+}
+
+// NewHealthcheck returns a new instance of healthcheck
+func NewHealthcheck(port uint16, logLevel log.Level) (*Healthcheck, error) {
+	var err error
+
+	healthcheck := &Healthcheck{
+		healthcheckPort: port,
+	}
+
+	log.SetLevel(logLevel)
+
+	healthcheck.client, err = rpc.NewClient(rpc.ConnectionModeWebsocket, rpc.WithAutoConfig(), rpc.WithBaseURL(&url.URL{
+		Scheme: "wss",
+		Host:   viper.GetString("hostname"),
```

Comment:
Does it make sense to always send the hostname config var? Or could that config option be optional, by default just using what's in the config?

Reply:
This is the same pattern we use in chia-exporter, defaults to localhost (since we assume its installed on the same host when getting the config, etc).